src/picknik_ur_base_config/objectives/looping_pick_and_place_object.xml

```diff
@@ -0,0 +1,95 @@
+<?xml version="1.0"?>
+<root BTCPP_format="4" main_tree_to_execute="Looping Pick and Place Object">
+    <!-- ////////// -->
+    <BehaviorTree ID="Looping Pick and Place Object">
```

Comment:
```suggestion
    <BehaviorTree ID="Looping Pick and Place Object" _description="Repeatedly attempt to detect and grasp a cuboid, then place it down">
```
Perhaps you have a better description, but this objective is the only one without